src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  width: $product-card-width;
+
+  padding: 32px 16px 16px;
+
+  box-sizing: border-box;
+  background-color: $background;
+
+  border: 1px solid $border-elements;
+  border-radius: 5px;
+
+  transition-duration: $transition-duration;
+
+  &__image {
+    display: block;
+
+    width: 160px;
+    height: 134px; // idk why, but tests fail by a single pixel when this is not hardcoded
+
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    color: $main;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    .card:hover & {
+      color: $card-title-on-hover;
+    }
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    color: $secondary;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-top: 16px;
+  }
+
+  &__reviews-count {
+    color: $main;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    color: $secondary;
+    font-size: 12px;
+    line-height: $price-line-height;
+  }
+
+  &__price-value {
+    color: $main;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: $price-line-height;
+  }
+
+  &__buy-button {
+    display: block;
+    box-sizing: border-box;
+
+    margin-top: 16px;
+
+    width: 100%;
+    height: $buy-button-height;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: $buy-button-height;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    border-radius: 5px;
+
+    transition-duration: inherit;
+
+    &:link,
+    &:visited {
+      color: $vibrant-accent-complementary;
+      background: $vibrant-accent;
+    }
+
+    &:hover,
+    &:active {
+      color: $vibrant-accent;
+      background: $vibrant-accent-complementary;
+      border: 1px solid $vibrant-accent;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
Cheklist:  All changes on :hover are smooth.
try always to add transition for :hover. It's needed for more user-friendly.